portal-docs/observability.md

```diff
@@ -32,12 +34,13 @@ Second, you will find an observe button on the left side of your application pag
 
 Once you reach the observability page, you will see a screen similar to the following. You use the view to
 
-1- Observe application metrics  
-2- Tracing requests  
-3- View low-code diagram along with observations
-4- View Logs  
-5- Navigate to the logs view  
-6- Navigate to the root cause analysis view  
+1. Observe application metrics  
+2. Tracing requests  
+3. View low-code diagram along with observations
+4. View Logs  
+5. Navigate to the logs view  
+6. Navigate to the root cause analysis view  
+
```

Comment:
Shall we change the numbering to bullet points since order does not seem to be relevant with regard to the listed items?
Also, better to change Tracing requests to Trace requests so that we follow the same verb tense across all listed items.